### PR TITLE
Add Cart Page

### PR DESCRIPTION
**Pull request by varun13169**
Add Cart Page

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The code seems fine to me....good to merge

**Review by DivyanshSareen — APPROVED** [review]

File: pages/cartPage/cartPage.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+                        <svg xmlns="http://www.w3.org/2000/svg"
+                            class="dui-inp-txt__icon-content dui-inp-txt__icon-content--search icon icon-tabler icon-tabler-search"
+                            width="56" height="56" viewBox="0 0 24 24" stroke-width="1.5" stroke="#059669" fill="none"
+                            stroke-linecap="round" stroke-linejoin="round">
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                            <circle cx="10" cy="10" r="7" />
+                            <line x1="21" y1="21" x2="15" y2="15" />
+                        </svg>
+                    </i>
+                    <input
```
Comment (itsmebipul00) [inline pages/cartPage/cartPage.html:35]: label tag

**Review by itsmebipul00 — APPROVED** [review]

File: pages/cartPage/cartPage.html (hunk 1 above)
Comment (varun13169) [reply pages/cartPage/cartPage.html:35]: I just merged it, will definitely make a follow-up PR to rectify it.

File: pages/cartPage/cartPage.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    width: fit-content;
+    min-width: 50rem;
+    height: fit-content;
+    padding: 2rem;
+    padding-left: 7.5rem;
+    padding-right: 7.5rem;
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+    background-color: white;
```
Comment (zoef-yt) [inline pages/cartPage/cartPage.css:49]: Use var for colours

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: pages/cartPage/cartPage.css (hunk 2 above)
Comment (varun13169) [reply pages/cartPage/cartPage.css:49]: Addressed the comment in the follow-up:
https://github.com/varun13169/gardenkart/pull/8

File: pages/cartPage/cartPage.html (hunk 1 above)
Comment (varun13169) [reply pages/cartPage/cartPage.html:35]: Addressed the comment in the follow-up:
https://github.com/varun13169/gardenkart/pull/8

**Pull request merged**